pkg/util/containerd/containerd_util.go

```diff
@@ -384,16 +367,7 @@ func (c *ContainerdUtil) IsSandbox(namespace string, ctn containerd.Container) (
 		return false, err
 	}
 
-	if labels["io.cri-containerd.kind"] == "sandbox" {
-		return true, nil
-	}
-
-	annotations, err := c.Annotations(namespace, ctn)
-	if err != nil {
-		return false, err
-	}
-
-	return annotations["io.kubernetes.cri.container-type"] == "sandbox", nil
+	return labels["io.cri-containerd.kind"] == "sandbox", nil
```

Comment:
Was the annotation always redundant or did this change in a recent containerd version?
If we remove the annotation check, we should update the references in the func documentation accordingly.

Reply:
Yes, normally both of them are set, as the `Annotations` are part of `Spec`, hence costly to parse, I'm removing it.